File: src/main/resources/static/js/service/applyService.js
app.service('applyService',function ($http) {
    this.findAll = function () {
        return $http.get('../apply/findAll');
    };
    this.updateFlag = function (apply){
        return $http.post('../apply/updateFlag',apply);
    };
    this.findNewApply10Size = function (){
        return $http.get('../apply/findNewApply10Size');
    };
});

File: src/main/java/com/shengxuan/speed/service/ApplyService.java
package com.shengxuan.speed.service;

import com.shengxuan.speed.entity.pojo.Apply;

import java.util.List;

public interface ApplyService {

    /**
     * 查询所有的10分钟内申请包括手动和程控组装成apply
     * @return
     */
    List<Apply> findAll();

    void updateFlag(Apply apply);

    List<Apply> findNewApply10Size();
}
